packages/components/src/lib/controllers/app-controller.js

```diff
@@ -1434,7 +1434,7 @@ export default class AppController {
           return
         }
       }
-      this._store.commit('app/setPrioritizeDefaultLanguage', false)
+      // this._store.commit('app/setPrioritizeDefaultLanguage', false)
```

Comment:
Should this be commented out?

Reply:
No, it is a fix for the language detection for the page - I removed it to fit to updated requirements.
But I decided not to delete it for now - I would delete after testing stage would be finished by Monica.